rosbag2_storage/test/rosbag2_storage/test_filesystem_helper.cpp

```diff
@@ -106,3 +106,23 @@ TEST(FilesystemHelper, get_file_name_returns_empty_string_if_passed_a_path_with_
 
   EXPECT_THAT(folder_name, Eq(""));
 }
+
+TEST_F(FilesystemHelperFixture, create_directory_passes_when_parent_exists)
+{
+  const auto path = FilesystemHelper::concat({temporary_dir_path_, "some_folder"});
+
+  EXPECT_TRUE(FilesystemHelper::create_directory(path));
```

Comment:
what's happening if that folder existed already? Say you invoke this command multiple times in a row?

Reply:
The test fixture should have deleted the temp folder on tear down.